pkg/kubectl/cmd/delete.go

```diff
@@ -167,6 +173,9 @@ func (o *DeleteOptions) Complete(f cmdutil.Factory, args []string, cmd *cobra.Co
 		o.WaitForDeletion = true
```

Comment:
As I noted in the other comment, this line becomes obsolete: `WaitForDeletion` is already `true` by default. And if user explicitly specifies `--wait=false`, it will be overwritten a few lines below.